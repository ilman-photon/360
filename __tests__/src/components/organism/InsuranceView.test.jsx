import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import InsuranceView from "../../../../src/components/organisms/InsuranceInformation/insuranceView";

describe("InsuranceView Components", () => {
  let container;
  beforeEach(() => {
    container = render(<InsuranceView />);
  });

  it("InsuranceView render", () => {
    expect(container).toMatchSnapshot();
  });
});
