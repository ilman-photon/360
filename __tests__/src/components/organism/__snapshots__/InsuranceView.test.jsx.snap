// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsuranceView Components InsuranceView render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1lwbda4-MuiStack-root"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Insurance Provider
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                />
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Plan Name
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                />
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Subscriber ID/Member ID
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                />
              </div>
            </div>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div
          class="css-1p5q5e5-MuiStack-root"
        >
          <div
            class="labelContainer"
          >
            <div
              class="label"
            >
              Group #
            </div>
          </div>
          <div
            class="css-1okj3ks-MuiStack-root"
          >
            <div
              style="margin-top: 10px;"
            />
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div>
          <h3
            class="MuiTypography-root MuiTypography-h3 css-1mie9p3-MuiTypography-root"
          >
            Policy Holder
          </h3>
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Are you the Subscriber?
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              />
            </div>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div
          class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  First Name
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                />
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Last Name
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                />
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Date of Birth
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  Invalid Date
                </div>
              </div>
            </div>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div
          class="css-1p5q5e5-MuiStack-root"
        >
          <div
            class="labelContainer"
          >
            <div
              class="label"
            >
              Relationship
            </div>
          </div>
          <div
            class="css-1okj3ks-MuiStack-root"
          >
            <div
              style="margin-top: 10px;"
            />
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div>
          <div
            class="MuiTypography-root MuiTypography-bodyRegular css-15atyh3-MuiTypography-root"
          >
            Upload images of your insurance.
          </div>
          <div
            class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27100%27%20height=%27183%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  alt=""
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27100%27%20height=%27183%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  alt=""
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div
          class="css-1p5q5e5-MuiStack-root"
        >
          <div
            class="labelContainer"
          >
            <div
              class="label"
            >
              Insurance Priority
            </div>
          </div>
          <div
            class="css-1okj3ks-MuiStack-root"
          >
            <div
              style="margin-top: 10px;"
            />
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div
          class="css-7mzo53-MuiStack-root"
        >
          <button
            class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root formButton outlined css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1b5efw0-MuiSvgIcon-root"
              data-testid="DeleteOutlineIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM8 9h8v10H8V9zm7.5-5-1-1h-5l-1 1H5v2h14V4z"
              />
            </svg>
            Remove Insurance
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <button
            class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root formButton primary css-sghohy-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="submit"
          >
            Edit
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1lwbda4-MuiStack-root"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Insurance Provider
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              />
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Plan Name
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              />
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Subscriber ID/Member ID
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              />
            </div>
          </div>
        </div>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="css-1p5q5e5-MuiStack-root"
      >
        <div
          class="labelContainer"
        >
          <div
            class="label"
          >
            Group #
          </div>
        </div>
        <div
          class="css-1okj3ks-MuiStack-root"
        >
          <div
            style="margin-top: 10px;"
          />
        </div>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div>
        <h3
          class="MuiTypography-root MuiTypography-h3 css-1mie9p3-MuiTypography-root"
        >
          Policy Holder
        </h3>
        <div
          class="css-1p5q5e5-MuiStack-root"
        >
          <div
            class="labelContainer"
          >
            <div
              class="label"
            >
              Are you the Subscriber?
            </div>
          </div>
          <div
            class="css-1okj3ks-MuiStack-root"
          >
            <div
              style="margin-top: 10px;"
            />
          </div>
        </div>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                First Name
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              />
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Last Name
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              />
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Date of Birth
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                Invalid Date
              </div>
            </div>
          </div>
        </div>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="css-1p5q5e5-MuiStack-root"
      >
        <div
          class="labelContainer"
        >
          <div
            class="label"
          >
            Relationship
          </div>
        </div>
        <div
          class="css-1okj3ks-MuiStack-root"
        >
          <div
            style="margin-top: 10px;"
          />
        </div>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div>
        <div
          class="MuiTypography-root MuiTypography-bodyRegular css-15atyh3-MuiTypography-root"
        >
          Upload images of your insurance.
        </div>
        <div
          class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27100%27%20height=%27183%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt=""
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-18wgjoz-MuiGrid-root"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27100%27%20height=%27183%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt=""
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
        </div>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="css-1p5q5e5-MuiStack-root"
      >
        <div
          class="labelContainer"
        >
          <div
            class="label"
          >
            Insurance Priority
          </div>
        </div>
        <div
          class="css-1okj3ks-MuiStack-root"
        >
          <div
            style="margin-top: 10px;"
          />
        </div>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="css-7mzo53-MuiStack-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root formButton outlined css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1b5efw0-MuiSvgIcon-root"
            data-testid="DeleteOutlineIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM8 9h8v10H8V9zm7.5-5-1-1h-5l-1 1H5v2h14V4z"
            />
          </svg>
          Remove Insurance
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root formButton primary css-sghohy-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="submit"
        >
          Edit
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
